docs/usage.md

```diff
@@ -214,7 +211,10 @@ If left unspecified, a default pattern is used: `data/*{1,2}.fastq.gz`
 
 ### `--single_end`
```

Comment:
```suggestion

```

Reply:
You can now delete all of the pipeline-specific parameter descriptions from this file because the docs will be auto-generated based on the JSON schema you have now added :)

Reply:
e.g. I have just kept in some information regarding the format of the input samplesheet but deleted all of the other parameters as you can see [here](https://github.com/nf-core/chipseq/blob/dev/docs/usage.md)